### PR TITLE
Return last document title together with history

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: c2corg_api/views/document.py [hunk 1]
```diff
@@ -433,6 +433,12 @@ def get(self):
         #    # validate permission (authenticated + associated)
         #    # return 403 if not correct
 
+        title = DBSession.query(DocumentLocale.title) \
+            .filter(DocumentLocale.document_id == id) \
+            .filter(DocumentLocale.culture == lang) \
```
Comment (asaunier) [inline c2corg_api/views/document.py]: Arf https://github.com/c2corg/v6_api/issues/101 :P

File: c2corg_api/views/document.py [hunk 2]
```diff
@@ -433,6 +433,12 @@ def get(self):
...
+            .order_by(DocumentLocale.id.desc()) \
```
Comment (asaunier) [inline c2corg_api/views/document.py]: Do you really need this? Isn't there only one version of the document in DocumentLocale?

**Comment by asaunier** [comment]
Small comments else looks good to me.

File: c2corg_api/views/document.py (hunk 2 above)
Comment (tsauerwein) [inline c2corg_api/views/document.py]: Right. `DocumentLocale` only contains the latest version.

File: c2corg_api/views/document.py [hunk 3]
```diff
@@ -433,6 +433,12 @@ def get(self):
         #    # validate permission (authenticated + associated)
         #    # return 403 if not correct
 
+        title = DBSession.query(DocumentLocale.title) \
```
Comment (tsauerwein) [inline c2corg_api/views/document.py]: This query might fail when you are requesting the history for a language without entry.

Comment (asaunier) [inline c2corg_api/views/document.py]: Good catch! The view should indeed raise an error if the requested language is not available.

Comment (gberaudo) [inline c2corg_api/views/document.py]: Currently I think it raises an internal error. Do we want a 404?

Comment (asaunier) [inline c2corg_api/views/document.py]: Yes

File: c2corg_api/views/document.py (hunk 2 above)
Comment (gberaudo) [inline c2corg_api/views/document.py]: Right, fixed.

File: c2corg_api/views/document.py (hunk 3 above)
Comment (gberaudo) [inline c2corg_api/views/document.py]: Done

**Comment by gberaudo** [comment]
Thanks for the review @asaunier and @tsauerwein. I think I have addressed all comments.

**Comment by asaunier** [comment]
:+1:

**Pull request merged**